core/services/ocr2/delegate.go

```diff
@@ -413,78 +411,40 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) ([]job.ServiceCtx, error) {
 			lggr,
 		)
 
-		// Create ORM layer
-		strategy := txmgr.NewQueueingTxStrategy(jobSpec.ExternalJobID, chain.Config().KeeperDefaultTransactionQueueDepth())
-		orm := keeper.NewORM(d.db, lggr, chain.Config(), strategy)
-
-		registryWrapper, err := keeper.NewRegistryWrapper(ethkey.MustEIP55Address(spec.ContractID), chain.Client())
-		if err != nil {
-			return nil, errors.Wrap(err, "unable to create keeper registry wrapper")
-		}
-
-		minIncomingConfirmations := chain.Config().MinIncomingConfirmations()
-		if cfg.MinIncomingConfirmations > 0 {
-			minIncomingConfirmations = cfg.MinIncomingConfirmations
-		}
-
 		jobSpec.KeeperSpec = &job.KeeperSpec{
 			ID:              spec.ID,
 			ContractAddress: ethkey.MustEIP55Address(spec.ContractID),
 			FromAddress:     ethkey.MustEIP55Address(spec.TransmitterID.String),
 		}
 
-		registrySynchronizer := keeper.NewRegistrySynchronizer(keeper.RegistrySynchronizerOptions{
-			Job:                      jobSpec,
-			RegistryWrapper:          *registryWrapper,
-			ORM:                      orm,
-			JRM:                      d.jobORM,
-			LogBroadcaster:           chain.LogBroadcaster(),
-			SyncInterval:             chain.Config().KeeperRegistrySyncInterval(),
-			MinIncomingConfirmations: minIncomingConfirmations,
-			Logger:                   lggr,
-			SyncUpkeepQueueSize:      chain.Config().KeeperRegistrySyncUpkeepQueueSize(),
-			NewTurnEnabled:           chain.Config().KeeperTurnFlagEnabled(),
-		})
-
-		oracle, err := libocr2.NewOracle(libocr2.OracleArgs{
+		rgstry, err := ocr2keeperschain.NewEVMRegistryV1_2(ethkey.MustEIP55Address(spec.ContractID).Address(), chain.Client())
```

Comment:
can we reuse registryWrapper here?
https://github.com/smartcontractkit/chainlink/blob/develop/core/services/keeper/delegate.go#L69

Reply:
I prefer not as most of it will be useless anyway since the API changed significantly for check upkeeps and the previous registries don't support OCR2. This way we can choose the registry version we want going forward and that registry is built into the new plugin repo.